packages/teleterm/src/ui/commandLauncher.ts

```diff
@@ -117,6 +117,18 @@ const commands = {
       }
     },
   },
+
+  'autocomplete.tsh-ssh': {
+    displayName: 'tsh ssh',
+    description: 'Run shell or execute a command on a remote SSH node',
+    run() {},
+  },
+  'autocomplete.tsh-proxy-db': {
+    displayName: 'tsh proxy db',
+    description:
+      'Start local TLS proxy for database connections when using Teleport',
+    run() {},
+  },
```

Comment:
Not sure if this is the best place for these commands to live, especially since their `run` functions will always be noops I think. But since for now we only have two, I think we might keep them here.